docs/multicall-notes.md

```diff
@@ -0,0 +1,225 @@
+
+For example, here's a query that will simply send ether from one address to another (with a state override that gives us the ETH initially):
+```json
+{
+	"jsonrpc": "2.0",
+	"id": 1,
+	"method": "eth_multicallV1",
+	"params": [
+		{
+			"blockStateCalls": [
+				{
+					"stateOverrides": {
+						"0xc000000000000000000000000000000000000000": {
+							"balance": "0x7d0"
+						}
+					},
+					"calls": [
+						{
+							"from": "0xc000000000000000000000000000000000000000",
+							"to": "0xc100000000000000000000000000000000000000",
+							"value": "0x3e8"
+						}
+					]
+				}
+			],
+			"traceTransfers": true
+		},
+		"latest"
+	]
+}
+```
+
+The output of this query is:
+```json
+{
+	"jsonrpc": "2.0",
+	"id": 1,
+	"result": [
+		{
+			"number": "0x4",
+			"hash": "0x859c932c5cf0dabf8d12eb2518e063966ac1a25e2fc49f1f02574a37f358d0b5",
+			"timestamp": "0x1f",
+			"gasLimit": "0x4c4b40",
+			"gasUsed": "0x5208",
+			"feeRecipient": "0x0000000000000000000000000000000000000000",
+			"baseFeePerGas": "0x2310a91d",
+			"prevRandao": "0x0000000000000000000000000000000000000000000000000000000000000000",
+			"calls": [
+				{
+					"returnData": "0x",
+					"logs": [
+						{
+							"address": "0x0000000000000000000000000000000000000000",
+							"topics": [
+								"0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+								"0x000000000000000000000000c000000000000000000000000000000000000000",
+								"0x000000000000000000000000c100000000000000000000000000000000000000"
+							],
+							"data": "0x00000000000000000000000000000000000000000000000000000000000003e8",
+							"blockNumber": "0x4",
+							"transactionHash": "0xa4d41019e71335f8567e17746b708ddda8b975a9a61f909bd3df55f4866cc913",
+							"transactionIndex": "0x0",
+							"blockHash": "0x859c932c5cf0dabf8d12eb2518e063966ac1a25e2fc49f1f02574a37f358d0b5",
+							"logIndex": "0x0",
+							"removed": false
+						}
+					],
+					"gasUsed": "0x5208",
+					"status": "0x1"
+				}
+			]
+		}
+	]
+}
+```
+
+Here the interesting part is:
+```json
+"address": "0x0000000000000000000000000000000000000000",
+"topics": [
+	"0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+	"0x000000000000000000000000c000000000000000000000000000000000000000",
+	"0x000000000000000000000000c100000000000000000000000000000000000000"
+],
+"data": "0x00000000000000000000000000000000000000000000000000000000000003e8",
+```
+As can be seen, the sending address is the zero address, `0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef` corresponds to signature `Transfer(address,address,uint256)`, `"0x000000000000000000000000c000000000000000000000000000000000000000"` corresponds the sending address and `0x000000000000000000000000c100000000000000000000000000000000000000` is the receiving address. The amount of ETH moved is stored in the `data` field.
```

Comment:
Maybe a note here that this is thus an "event"/log which thus has two topics (sender, receiver) and the value (the data field)